backend/src/@types/fastify.d.ts

```diff
@@ -16,6 +16,7 @@ import { TEventBusService } from "@app/ee/services/event/event-bus-service";
 import { TServerSentEventsService } from "@app/ee/services/event/event-sse-service";
```

Comment:
IGNORE PLACEMENT

<img width="2632" height="740" alt="Image" src="https://github.com/user-attachments/assets/7d6b291c-6811-4aff-86f2-24ff6ab9daaa" />

I think we can also place the version information here